client/src/components/TermsAndConditionsModal.js

```diff
@@ -0,0 +1,308 @@
+import React, { useState } from "react";
+import { createUseStyles } from "react-jss";
+import { withRouter } from "react-router-dom";
+import { useHistory } from "react-router";
+import Modal from "react-modal";
+import Button from "./Button/Button";
+import PropTypes from "prop-types";
+
+const useStyles = createUseStyles({
+  modal: {
+    "& h2": {
+      fontSize: "25px",
+      lineHeight: "31px",
+      fontWeight: "bold",
+      textAlign: "center",
+      marginBottom: "30px",
+      "& img": {
+        margin: "0 6px 0 0",
+        verticalAlign: "middle"
+      }
+    },
+    "& p": {
+      fontSize: "16px",
+      lineHeight: "25px",
+      textAlign: "left",
+      "& img": {
+        margin: "4px 12px 12px 0"
+      }
+    },
+    "& .smaller": {
```

Comment:
`smaller` should be a key name and not nested under modal. you can use it in your code by calling classes.smaller as the classname